playbooks/upscaling_pre-tasks.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# Check that new number of nodes is higher
+- hosts: masters
+  tasks:
+  - name: Get number of nodes
+    shell: oc get nodes | grep app-node -c
+    register: oc_old_result
+  - name: Set new number of nodes
+    set_fact:
+      # Number of nodes in the new configuration (incremented by 1 by default)
+      nodes: "{{ oc_old_result.stdout | int + 1 }}"
+  - name: Check that new value is higher
+    assert:
+      that: 'nodes | int > oc_old_result.stdout | int'
+      msg: >
+        FAIL: New number of nodes must be higher than the current one
+        ({{ oc_old_result.stdout | int }}).
+
+# Update openstack_num_nodes value in the inventory
+- hosts: localhost
+  tasks:
+  - name: Replace openstack_num_nodes value
+    lineinfile:
+      path: ../../{{ inv_directory }}/group_vars/all.yml
+      regexp: '^openstack_num_nodes:'
+      line: 'openstack_num_nodes: {{ hostvars[groups["masters"][0]]["nodes"] }} '
```

Comment:
the up-scaling increments current number of nodes 1 ONLY if user does not override it by defining a constant higher than that...so if I removed it and only incremented it by 1 per deployment, it would take several hours for someone to scale it up by...say 10, wouldn't it?